### DISCUSSION TITLE
Cleaner Circuit

### DISCUSSION THREAD

**Discussion by orrious** [tagyoureit/nodejs-poolController#538] [Nixie] [answered]
I created: 
A circuit named `Cleaner` type `Master Cleaner`
A valve called `vCleaner` and assigned circuit to `Cleaner`
A pump circuit and assigned circuit to `Cleaner` (Note the RPM is 2000 where Pool circuit is 2500)
A schedule to run circuit `Cleaner` from 8am - 11am

I am unable to run the `Cleaner` circuit without the `Pool` circuit running.  The RPM is at 2500 for the `Pool`, not 2000 for the `Cleaner`  The schedule will also not run.

**Comment by rstrouse** [comment, marked as answer]
Master cleaner circuit functions are very much a pool mode operation.   It has special processing wrt delays, heater interactions, and booster pumps.   If you are not connecting this circuit to a booster pump or need it to manipulate standard pool operations it is not the right circuit function for your cleaner circuit.

If this is simply to rotate a valve and set a speed on the pump then Master Cleaner is not for you.

**Reply by orrious** [reply]
That's what I needed to know.  Thanks!
